srcpkgs/Quaternion/template

```diff
@@ -1,19 +1,21 @@
 # Template file for 'Quaternion'
 pkgname=Quaternion
-version=0.0.9.4e
+version=0.0.95.beta4
```

Comment:
Why move to beta? We prefer to stick with stable releases.

Reply:
v0.0.9.4 is almost 2 years old

Reply:
0.0.9.4f marked as "latest" is just two months, use it if being new is main reason to prefer unstable thing.

Reply:
There's no reason to update to 0.0.9.4f since it has [the same code](https://github.com/quotient-im/Quaternion/compare/0.0.9.4e...0.0.9.4f) as 0.0.9.4e (which is from April 2020). It is just a rebuild for windows to use newer Qt: https://github.com/quotient-im/Quaternion/releases/tag/0.0.9.4f